graph/patterns/alternate-key.md

```diff
@@ -0,0 +1,48 @@
+# Alternate Key Pattern
+
+Microsoft Graph API Design Pattern
+
+*The Alternate Key Pattern provides the ability to query for a single, specific entity identifiable through an alternative attribute (called key) that is not its unique identifier*
+
+## Problem
+--------
+
+All entities in our system are identified by an UUID - which guarantees uniqueness. Often though, that same entity can also be uniquely identified by an alternative, more convenient attribute.
```

Comment:
UUID???
If we do use acronyms, we need to define them.

Reply:
I will expand it, I though [UUID ](https://en.wikipedia.org/wiki/Universally_unique_identifier)was common enough to avoid the expansion